css/styles.css

```diff
@@ -37,3 +38,49 @@ button{
   color: white;
   background-color: rgb(105, 37, 105);
 }
+
+.margin-top{
+  margin-top: 40px;
+}
+
+.entry-form{
+  display: none;
+}
```

Comment:
What is this for? You should be using my `hidden` class I provided for you in `layout.css`